### ISSUE TITLE
Attention visualization

### ISSUE THREAD

**Issue by swordLJP** [youqingxiaozhua/APViT#3]
I want to get attention visualization of different expressions like whitch shown in your paper Transfer, can you provide some tools? I am new to Paddle.  It's a bit difficult to convert the code from pytorch to paddle.

**Comment by youqingxiaozhua** [comment]
As we cited in the paper, we used [this tool](https://github.com/hila-chefer/Transformer-Explainability) to visualize the attention maps. If you use pytorch for model training, it may not be necessary to convert it to paddle. Their APIs are very similar, and our implementation follows the original ViT codes as in timm.

**Comment by ilanaveh** [comment]
Hello, I wanted to ask for a bit more information regarding the attention visualization.
In the method you're using, attention can be visualized relative to different classes. Were the images you show in the paper correctly classified by the different models? and if not, did you use the predicted or the true label for visualization? Also, if you can provide the visualization code that would be extremely helpful, since when using the code published by the original paper (Chefer 2021), it's not trivial to incorporate the attention-pooling modules with their LRP model. Thank you very much!
